package.json

```diff
@@ -104,8 +104,7 @@
 		"lint": "eslint . --ext .js",
 		"lint:fix": "npm run lint -- --fix",
 		"xcframework:setup": "./bin/xcframework-setup",
-		"xcframework:build": "npm run xcframework:setup && pushd ./ios-xcframework && ./build.sh && popd",
-		"version": "npm run bundle && git add -A bundle"
```

Comment:
I was curious of the origin of this script. It appears to have been added in https://github.com/wordpress-mobile/gutenberg-mobile/pull/255 as a way to ease remembering to bundle prior to committing a version bump. Given that, removing the script makes sense now that we bundle on the CI server and automate version bumps with other scripts.